help/destinations/catalog/advertising/pubmatic.md

```diff
@@ -0,0 +1,129 @@
+| ---------------- | ------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Export type      | **[!UICONTROL Segment export]** | You are exporting all members of a segment (audience) with the identifiers (name, phone number, or others) used in the PubMatic Connect destination.                                                                                                                                                                               |
+| Export frequency | **[!UICONTROL Streaming]**      | Streaming destinations are "always on" API-based connections. As soon as a profile is updated in Experience Platform based on segment evaluation, the connector sends the update downstream to the destination platform. Read more about [streaming destinations](/help/destinations/destination-types.md#streaming-destinations). |
+
+{style="table-layout:auto"}
+
+## Connect to the destination {#connect}
+
+> [!IMPORTANT]
+>
+> To connect to the destination, you need the **[!UICONTROL Manage Destinations]** [access control permission](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+To connect to this destination, follow the steps described in the [destination configuration tutorial](../../ui/connect-destination.md). In the configure destination workflow, fill in the fields listed in the two sections below.
+
+### Authenticate to destination {#authenticate}
+
+To authenticate to the destination, fill in the required fields and select **[!UICONTROL Connect to destination]**.
+
+![How to authenticate](/help/destinations/assets/catalog/advertising/pubmatic/authenticate-destination.png)
+
+- **[!UICONTROL Bearer token]**: Fill in the bearer token to authenticate to the destination.
+
+### Fill in destination details {#destination-details}
+
+To configure details for the destination, fill in the required and optional fields below. An asterisk next to a field in the UI indicates that the field is required.
+
+![Destination details](/help/destinations/assets/catalog/advertising/pubmatic/destination-details.png)
+
+- **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
+- **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+- Data partner ID: The data partner ID set up in you PubMatic account for this integration.
+- Default Country Code: The default country code that should be applied to all identities if none is provided in the profile.
```

Comment:
Please maintain uniform capitalization of UI elements. **Default country code** instead of **Default Country Code**.

You'll need to update this in the destination configuration too.